gtda/images/preprocessing.py

```diff
@@ -201,3 +201,115 @@ def transform(self, X, y=None):
         Xt = np.concatenate(Xt)
 
         return Xt
+
+
+@adapt_fit_transform_docs
+class ImageToPointCloud(BaseEstimator, TransformerMixin):
+    """Transformer returning a collection of point clouds that are the
```

Comment:
Missing a one-line description. How about "Represent 2D/3D binary images as point clouds of the active pixels."

Reply:
I am not sure what you mean. For now all images/preprocessing module only have one paragraph description as they are all quite simple.

Reply:
This is because of a sloppiness in reviewing on my part. In fact we do have short (one or two line) descriptions virtually everywhere else. It is important to have them for sphinx documentation, as sphinx always print the first paragraph (whatever it is) in `index.html`. This makes for a rather messy result unless we stick with very short first paragraphs.

Reply:
Understood.

Reply:
My original suggestion for this one-liner wasn't great but the current is a little too long I feel. How about "Represent active pixels in 2D/3D binary images as points with coordinates given by their array indices."?